src/main/java/uk/gov/beis/els/categories/dishwashers/model/DishwashersForm.java

```diff
@@ -2,47 +2,87 @@
+@GroupSequenceProvider(DishwashersFormSequenceProvider.class)
 public class DishwashersForm extends StandardTemplateForm30Char {
 
-  @FieldPrompt("Energy efficiency class indicator")
-  @NotBlank(message = "Select an energy efficiency indicator", groups = {Default.class, InternetLabellingGroup.class})
+  @FieldPrompt("What style of label do you need to generate?")
+  @NotBlank(message = "Specify the style of label you need to generate", groups = {Default.class, InternetLabellingGroup.class})
   @DualModeField
-  private String efficiencyRating;
+  private String applicableLegislation;
+
+  @FieldPrompt("Rated capacity (number of standard place settings) for the standard cleaning cycle")
+  @Digits(integer = 2, fraction = 0, message = "Enter a rated capacity, up to 2 digits long")
+  private String capacity;
+
+  @FieldPrompt("Airborne acoustical noise emissions expressed in dB(A) re 1 pW")
+  @Digits(integer = 2, fraction = 0, message = "Enter the noise emission, up to 2 digits long")
+  private String noiseEmissions;
 
+  // Pre march 2021 fields
   @FieldPrompt("Annual energy consumption (AEC) in kWh per year")
-  @Digits(integer = 3, fraction = 0, message = "Enter an annual energy consumption, up to 3 digits long")
+  @Digits(integer = 3, fraction = 0, message = "Enter an annual energy consumption, up to 3 digits long", groups = PreMarch2021Field.class)
   private String annualEnergyConsumption;
 
   @FieldPrompt("Annual water consumption (AWC) in litres per year")
-  @Digits(integer = 4, fraction = 0, message = "Enter an annual water consumption, up to 4 digits long")
+  @Digits(integer = 4, fraction = 0, message = "Enter an annual water consumption, up to 4 digits long", groups = PreMarch2021Field.class)
   private String annualWaterConsumption;
 
-  @FieldPrompt("Rated capacity (number of standard place settings) for the standard cleaning cycle")
-  @Digits(integer = 2, fraction = 0, message = "Enter a rated capacity, up to 2 digits long")
-  private String capacity;
-
   @FieldPrompt("Drying efficiency class")
-  @NotBlank(message = "Select a drying efficiency class")
+  @NotBlank(message = "Select a drying efficiency class", groups = PreMarch2021Field.class)
   private String dryingEfficiencyRating;
 
-  @FieldPrompt("Airborne acoustical noise emissions expressed in dB(A) re 1 pW")
-  @Digits(integer = 2, fraction = 0, message = "Enter the noise emission, up to 2 digits long")
-  private String noiseEmissions;
+  // Post march 2021 fields
+  @FieldPrompt("Link to the EPREL or other website which provides further energy efficiency information about this product")
+  @Pattern(regexp = "^(https|http)://([a-zA-Z0-9]+)\\.[a-zA-Z0-9]+", message = "Enter a Link to the EPREL or other website. Links must start with http:// or https:// and contain at least one dot (.) character", groups = PostMarch2021Field.class)
```

Comment:
I've just done a simple-ish regex check for a valid scheme + some kind of TLD. Not sure if the validation message is clear enough? I didn't want to use terms like scheme or TLD, but its pretty verbose to explain it fully without them.

Reply:
I'll move this field into the generic base form as well later, since it will be used by all the new labels.

Reply:
The first `[a-zA-Z0-9]` needs to allow hyphens as they're valid in the domain name - at the moment this would reject URLs such as `https://example-domain.com`. Other than that it should be fine. I think the validation message is clear (though Link needs a lowercase `l`)

Reply:
Ah good point. Will update